src/components/RegisterLanding.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import StyledFieldset from './StyledFieldset';
+
+export default function RegisterLanding({ setRegisterStep }) {
+  const [isRegistered, setIsRegistered] = useState(null);
+
+  const handleRadioChoice = (e) => {
+    const boolValue = e.target.value === 'yes';
+    setIsRegistered(boolValue);
+  };
+
+
+  return (
+    <>
+     
+
```

Comment:
Can you please check your [prettier](https://prettier.io/) settings? 
The code formatting in these files needs to be fixed.
Please either turn on auto-formatting or run the `format` script.